R/DataDistribution.R

```diff
@@ -55,17 +75,30 @@ setGeneric("cumulative_distribution_function", function(dist, x, n, theta, ...)
 #' @exportClass Normal
 setClass("Normal", representation(
     two_armed = "logical"
-    ),
-    contains = "DataDistribution")
+),
+contains = "DataDistribution")
+
 
-#' @param two_armed s. slot
+#' @param two_armed logical indicating if a two-armed trial is regarded
+#'
+#' @examples
+#' datadist <- Normal(two_armed = TRUE)
+#'
+#' @return An object of class \code{Normal}
```

Comment:
Debatable, but I find documenting the return values of functions not always necessary (especially when you have multiple in one file since it is not straight forward to tell which one goes with which). Imho totally unnecessary with constructors since it is pretty obvious they just return an object of the corresponding type ;)

Reply:
Agree